Sources/SwiftyMonaco/MonacoViewController.swift

```diff
@@ -68,14 +68,30 @@ public class MonacoViewController: ViewController, WKUIDelegate, WKNavigationDel
     #endif
     
     public func webView(_ webView: WKWebView, didFinish navigation: WKNavigation!) {
+        // Syntax Highlighting
+        let syntax = self.delegate?.monacoView(getSyntax: self)
+        let syntaxJS = syntax != nil ? """
+        // Register a new language
+        monaco.languages.register({ id: 'mySpecialLanguage' });
+
+        // Register a tokens provider for the language
+        monaco.languages.setMonarchTokensProvider('mySpecialLanguage', (function() {
+            \(syntax!.configuration)
+        })());
+        """ : ""
+        let syntaxJS2 = syntax != nil ? ", language: 'mySpecialLanguage'" : ""
+        
+        // Code itself
         let text = self.delegate?.monacoView(readText: self) ?? ""
         let b64 = text.data(using: .utf8)?.base64EncodedString()
         let javascript =
         """
         (function() {
-          editor.create({value: atob('\(b64 ?? "")'), automaticLayout: true, theme: "\(detectTheme())"});
-          var meta = document.createElement('meta'); meta.setAttribute('name', 'viewport'); meta.setAttribute('content', 'width=device-width'); document.getElementsByTagName('head')[0].appendChild(meta);
-          return true;
+        \(syntaxJS)
+
+        editor.create({value: atob('\(b64 ?? "")'), automaticLayout: true, theme: "\(detectTheme())"\(syntaxJS2)});
+        var meta = document.createElement('meta'); meta.setAttribute('name', 'viewport'); meta.setAttribute('content', 'width=device-width'); document.getElementsByTagName('head')[0].appendChild(meta);
+        return true;
```

Comment:
We need to do something with this in the future because it's completely insecure